tests/integration/standard/test_query.py

```diff
@@ -1251,6 +1250,7 @@ def test_setting_keyspace_and_same_session(self):
 
 @greaterthanorequalcass40
 class SimpleWithKeyspaceTests(QueryKeyspaceTests, unittest.TestCase):
+    @unittest.skip
```

Comment:
Why are we skipping this now?

Reply:
Specifying the keyspace before V5 is allowed right now and raising an exception would break the API. The keyspace is ignored bu there may be people doing it (like us in CQLEngine), I added that as part of PYTHON-504, we can remove the skip after that.